api/shape/dxf.js

```diff
@@ -0,0 +1,43 @@
+import { fromDxf, toDxf } from '@jsxcad/convert-dxf';
+import { getSourceLocation, read, write } from '@jsxcad/sys';
+import { gridView, qualifyViewId } from './view.js';
+
+import { Shape } from './Shape.js';
+import { ensurePages } from '@jsxcad/geometry';
+
+export const LoadDxf = Shape.registerMethod3(
```

Comment:
Does this add support for loading and exporting DXF files?

Reply:
Mostly it moves it into the core and updates the interface a bit.

Not all of DXF is supported, though -- needs support for SPLINE, at least.

Reply:
DXF seems like a remarkably fragile format given how widely supported it is. I'm had tons of issues with different version of DXF not playing well together